src/PIL/ImageGrab.py

```diff
@@ -70,15 +69,15 @@ def grab(
                 left, top, right, bottom = bbox
                 im = im.crop((left - x0, top - y0, right - x0, bottom - y0))
             return im
-    xdisplay = cast(Union[str, None], xdisplay)  # type: ignore[redundant-cast, unused-ignore]
+    display_name: str | None = xdisplay
```

Comment:
```suggestion
    # Cast to Optional[str] needed for Windows and macOS.
    display_name: str | None = xdisplay
```
Shall we at least add an explanatory comment for why we are assigning a variable to a different variable with the same type hint?